backend/recipe/GroceryList.py

```diff
@@ -60,8 +60,11 @@ def delete_item(id, item):
                     continue
                 else:
                     shaved_grocery_list.append(existing_item)
-            final_list = items_string = ', '.join(shaved_grocery_list)
-            cursor.execute('UPDATE grocery_list SET grocery_list = ? WHERE usr_id = ?', (final_list, id))
+            final_list = ', '.join(shaved_grocery_list)
+            if final_list: 
```

Comment:
why cant we just leave empty list, does it break?

Reply:
Yes. If left empty, the frontend renders an empty bullet point with a delete button that can mess things up. The add recipe or grocery or stock item functions create a new list if there's no list associated with that usr_id, so this works.